.env.example

```diff
@@ -101,6 +101,8 @@ STRIPE_TEAM_MONTHLY_PRICE_ID=
 STRIPE_WEBHOOK_SECRET=
 STRIPE_PRIVATE_KEY=
 STRIPE_CLIENT_ID=
+STRIPE_PAYMENT_FEE_FIXED=
```

Comment:
@zomars I think we can remove thse variables from here. They should be part of .appStore.example only right?

Reply:
@zomars If you let me know what to do here, happy to push the commits.